widgets/src/main/java/org/pentaho/gwt/widgets/client/tabs/PentahoTabPanel.java

```diff
@@ -98,9 +102,15 @@ public void selectTab( PentahoTab selectedTab ) {
       if ( tab == selectedTab ) {
         tab.setSelected( true );
         tabDeck.showWidget( tabDeck.getWidgetIndex( tab.getContent() ) );
+        tabDeck.getWidget( tabDeck.getWidgetIndex( tab.getContent() ) ).removeStyleName( "is-hidden" );
+        tab.getElement().setTabIndex( 0 );
+        tab.getElement().focus();
```

Comment:
This would better go through the `PentahoTab` API, which should implement `Focusable`, and would then expose the `setTabIndex` and `setFocus` methods.

Reply:
what are we going to achieve out of implementing Focusable ? as the code is already working and unit tested as per the standard. Implementing Focusable is an option, but is it must ? really ? needed ???

Reply:
this has been handled by PentahoTab.java

Reply:
Implementing `Focusable` is useful, at least, for dialogs, as it is what is used to identify keyboard focusable elements in which to place the focus initially. If this component is not currently being used inside a dialog, we can skip this.